site/archive.md

```diff
@@ -0,0 +1,92 @@
+
+Note that file resources that already exist on the target system need not be included in the archive, provided that the model specifies the correct location on the target system.
+
+- [Example](#example)
+- [Archive Structure](#archive-structure)
+- [Using Multiple Archive Files](#using-multiple-archive-files)
+
+## Example
+
+This example shows an application with a `SourcePath` value referencing an EAR file resource contained in the archive.
+
+```yaml
+appDeployments:
+    Application:
+        simpleear :
+            SourcePath: wlsdeploy/applications/simpleear.ear
+            Target: 'my-cluster'
+            ModuleType: ear
+```
+
+The example above shows the attribute `SourcePath` of the `simpleear` application with a value of `wlsdeploy/applications/simpleear.ear`.  The prefix `wlsdeploy/` indicates that the resource is located in the archive file in the specified location, and will be deployed to that directory in the domain, in this case `<domain-home>/wlsdeploy/applications/simpleear.ear`.
+
+## Archive Structure
+
+These are the paths within the archive that are used for different types of resources. Users can create further directory structures underneath these locations to organize the files and directories as they see fit.  
+
+### `atpwallet`
+
+The directory where a wallet can be stored for use with Oracle Autonomous Transaction Processing Cloud Database. The file resource name is not specified in the model, and is assumed to be a single ZIP file in the archive at this location.  
+
+### `model`
+The directory where the model is optionally located. Only one model file, either in YAML or JSON, is allowed, and it must have the appropriate YAML or JSON file extension.
+
+### `opsswallet`
+
+The directory where a wallet can be stored for use with Oracle Platform Security Services. The file resource name is not specified in the model, and is assumed to be a single ZIP file in the archive at this location.  
+
+### `wlsdeploy/applications`
+The root directory under which applications are stored. Applications can be stored in the archive as EAR or WAR files, or expanded* under this folder. A sample expanded WAR application might have these entries:
+```
+wlsdeploy/applications/myApp/index.jsp
+wlsdeploy/applications/myApp/META-INF/MANIFEST.MF
+wlsdeploy/applications/myApp/WEB-INF/classes/MyClass.class
+wlsdeploy/applications/myApp/WEB-INF/web.xml
+wlsdeploy/applications/myApp/WEB-INF/weblogic.xml
+```
+
+\* Expanded application directories are supported after WebLogic Deploy Tooling release 1.6.2
+
+### `wlsdeploy/classpathLibraries`
+The root directory under which JARs/directories used for server classpaths are stored. Every file resource under this directory is extracted, even those not referenced in the model. 
```

Comment:
JARs/directories -> `JARs/directories` (code font)

Reply:
Changed to:
JARs and directories
without code tags?

Reply:
JARs/directories in code font means putting the text inside back ticks. Then it looks like this, `JARs/directories`

Reply:
I changed it to:
under which JAR files and directories used
I think that doesn't need code tags?

Reply:
Yes, that text does not need to be in code font. (PS I am requesting code font, not tags, for the directories).